ReactWindows/ReactNative/Modules/Vibration/VibrationModule.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System;
+using ReactNative.Bridge;
+using Windows.Phone.Devices.Notification;
+
+namespace ReactNative.Modules.Vibration
+{
+    /// <summary>
+    /// Represents the module for vibration.  Note this is only available on Windows Mobile.
+    /// </summary>
+    public class VibrationModule : NativeModuleBase
+    {
+        private readonly bool _isMobile;
+
+        internal VibrationModule() : this(IsMobile())
+        {
+        }
+
+        internal VibrationModule(bool isMobile)
+        {
+            _isMobile = isMobile;
+        }
+
+        private static bool IsMobile()
+        {
+            var platformFamily = Windows.System.Profile.AnalyticsInfo.VersionInfo.DeviceFamily;
```

Comment:
Fixed as I moved the logic into the `WindowsPlatformHelper` which has a `DeviceFamily` which contains all of the possible devices for UWP.